nodejs/week2/homework/meal-sharing/src/backend/api/reviews-router.js

```diff
@@ -0,0 +1,28 @@
+const express = require("express");
+const router = express.Router();
+
+const reviews = require("./../data/reviews.json");
+
+router.get("/", async (request, response) => {
+  try {
+    console.log(reviews);
+    console.log("in /api/reviews");
+  } catch (error) {
+    throw error;
+  }
+});
+router.get("/:id", async (request, response) => {
+  try {
+    console.log(reviews);
+    console.log("in /api/reviews");
+    const id = parseInt(request.params.id);
+    const selected = reviews.filter(review => review.id === id);
+    response.send(selected);
+  } catch (error) {
+    throw error;
```

Comment:
throwing an error here is not helpful because it would crash your server app if it's not caught.
you can return an error code like` 500 Internal Server Error instead`